app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,28 @@
+class InvitesController < ApplicationController
+  def new; end
+
+  def create
+    if !handle_in_system?
+      flash[:error] = "The Github user you selected doesn't have an email address associated with their account."
+    else
+      @invitee_handle = params[:github_handle]
+      @user = current_user
+      InvitationMailer.with(user: @user, invitee_handle: @invitee_handle).invitation_email.deliver_now
```

Comment:
Thank you for all your work with the registration email. It was quite easy to follow

Reply:
I aims to please, and by please I mean write code that is easy to understand! Also you are very smart, so there's that.